cadence/contracts/Recipe.cdc

```diff
@@ -0,0 +1,516 @@
+import "FungibleToken"
+import "NonFungibleToken"
+import "Burner"
+
+/// NB: This contract is no longer supported. NFT Storefront V2 is recommended
```

Comment:
@franklywatson I noticed this recipe is using the old `NFTStorefront` contract (the recipe contract code was essentially a copy paste of the entire `NFTStorefront` contract source code) , thoughts on updating to use the V2 contract?

Reply:
Yeah we should not use V1 and if we're copying the entire contract let's used the deployed version to generate a dependency and not have the code copied in full into the project.

Reply:
@franklywatson will do - it is a quick change on the front-end to make the contract.cdc/Recipe.cdc code snippet optional -  I think it makes sense to just delete the `Recipe.cdc` code altogether where it is just a copy of a deployed contract and leave only the transaction code (same applies to some of the TS recipes)